design/comments.go

```diff
@@ -152,6 +152,10 @@ var _ = a.Resource("work-item-comments", func() {
 			a.GET("comments"),
 		)
 		a.Description("List comments associated with the given work item")
+		a.Params(func() {
+			a.Param("page[offset]", d.String, "Paging start position")
```

Comment:
> "Paging start position"

To be easily consumable by UI and other folks, please specify range information. https://en.wikipedia.org/wiki/Interval_(mathematics)#Including_or_excluding_endpoints

![bildschirmfoto von 2017-01-23 19-38-35](https://cloud.githubusercontent.com/assets/193408/22217521/9752dbf4-e1a3-11e6-9c96-1e92630a4499.png)

Does paging start at 0 or at 1, for example? Please also give an example and don't hesitate to write a multiline description using 

```go
`multiline
strings
with
backticks`
```

As our API grows, we should have a pretty decent documentation.

If pagination works the same all over the place, create a string constant and re-use it. The same applies to the limit parameter.